docs/docs/cmd/context/option/option-set.md

```diff
@@ -0,0 +1,31 @@
+# context option set
+
+Allows to add a new key to the context with value
+
+## Usage
+
+```sh
+m365 context option set [options]
+```
+
+## Options
+
+`-n, --name <name>`
+: The option name for which to define the value
+
+`-v, --value <value>`
+: Default value for the option
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Adds a new name to the CLI for Microsoft 365 context in the current working folder
```

Comment:
Updated the example description since we have changed the specs for better clarity